topic02-schedule/book-schedule/01.Week-1.md

```diff
@@ -0,0 +1,9 @@
+## Week 1: Induction
```

Comment:
I'm wondering if we might be better structuring the schedule by month rather than by week, as we will run out of space for weeks across the top. If we have a tab for each month, we can have the weeks for that month in that tab